module/x/gravity/types/msgs.go

```diff
@@ -593,3 +594,27 @@ func (msg *MsgCancelSendToEth) GetSigners() []sdk.AccAddress {
 	}
 	return []sdk.AccAddress{sdk.AccAddress(acc)}
 }
+
+// MsgSubmitBadSignatureEvidence
+// ======================================================
+
+// ValidateBasic performs stateless checks
+func (e *MsgSubmitBadSignatureEvidence) ValidateBasic() error {
+	return nil
+}
+
+// GetSignBytes encodes the message for signing
+func (msg MsgSubmitBadSignatureEvidence) GetSignBytes() []byte {
+	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(msg))
```

Comment:
no amino please

Reply:
All of the other functions in the file are using `ModuleCdc.MustMarshalJSON`. It doesn't seem like a good idea to change it for just one function, and refactoring everything would be out of scope for this PR